Content.Server/Body/BodyCommands.cs

```diff
@@ -144,4 +149,92 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
             shell.SendText(player, $"No mechanism was found with name {mechanismName}.");
         }
     }
+
+    class HurtCommand : IClientCommand
+    {
+        public string Command => "hurt";
+        public string Description => "Ouch";
+        public string Help => $"Usage: {Command} <type> <amount> (<entity uid/_>) (<ignoreResistance>)";
+
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            // Check if we have enough for the dmg types to show
+            if (args.Length < 1)
+            {
+                shell.SendText(player, Help);
+                return;
+            }
+
+            bool fallback = false;
+            DamageType dmgType = default;
+            // Send all damage types if we can't parse (e.g. hurt ?)
+            if (!Enum.TryParse<DamageClass>(args[0], true, out var dmgClass))
+            {
+                // Fallback to DamageType
+                if (!Enum.TryParse(args[0], true, out dmgType))
+                {
+                    var msg = "";
+                    foreach (var dClass in Enum.GetNames(typeof(DamageClass)))
+                    {
+                        msg += $"\n{dClass}";
+                        var types = Enum.Parse<DamageClass>(dClass).ToTypes();
+                        foreach (var dType in types)
+                        {
+                            msg += $"\n - {dType}";
+                        }
+                    }
+                    shell.SendText(player, $"Damage Types:{msg}");
+                    return;
+                }
+                // Remember that we use DamageType not DamageClass
+                fallback = true;
+            }
```

Comment:
You could move all of this to the end of the method and do a normal `if damageclass parse, else if damagetype parse, else error message`.